src/Generators/AbstractTestsGenerator.php

```diff
@@ -172,8 +172,8 @@ protected function getMockModel($model): array
             return [];
         }
 
-        $modelClass = $this->getModelClass($model);
-        $factory = $modelClass::factory();
+        $factoryNamespace = "{$this->getOrCreateNamespace('factories')}\\{$model}Factory";
```

Comment:
as we are using the HasFactory trait - `$modelClass::factory();` is valid factory getter and it makes the same

Reply:
$modelClass::factory() throws this error - https://github.com/RonasIT/laravel-entity-generator/issues/100#issuecomment-2527188851 when we use a non-default directory for the model class.

Reply:
@pirs1337 git it! Let's remove HasFactory trait of the model as it became useless at all